packages/components/Molecules/Card/CardHonourRoll.vue

```diff
@@ -9,7 +9,7 @@
         <span v-if="inductionYear && category" class="rpl-card-honour-roll__separator">|</span>
         <span v-if="category">{{ category }}</span>
       </p>
-      <p class="rpl-card-honour-roll__summary" v-if="summary" v-html="summary" />
+      <div class="rpl-card-honour-roll__summary" v-if="summary" v-html="summary"></div>
```

Comment:
Or just render it like `<p...>{{ summary }}</p>`?

Reply:
@rubyllamadora `<p...>{{ summary }}</p>` won't render HTML, it will be encoded.

Reply:
Got it, there's html inside. I thought it was pure text. Summary field in BE must be using wysiwyg

Reply:
> Got it, there's html inside. I thought it was pure text. Summary field in BE must be using wysiwyg

Yes